src/QirRuntime/samples/StandaloneInputReference/.clang-tidy

```diff
@@ -0,0 +1,3 @@
+Checks: 
```

Comment:
As commented previously, would it be possible to avoid introducing a different clang-tidy file but instead filter out the header with --header-filter and/or --line-filter settings?

Reply:
Used `InheritParentConfig: true` in the new file.

Unfortunately `--line-filter` is not supported in the configuration file, so including headers in clang-tidy checks is opt-in through `HeaderFilterRegex`.